modules/Kconfig.nordic

```diff
@@ -20,6 +21,16 @@ menuconfig NRF_802154_RADIO_DRIVER
 
 if NRF_802154_RADIO_DRIVER
 
+config NRF_802154_MULTIPROTOCOL_SUPPORT
+	bool
+	# Hidden option
+	default n
```

Comment:
Options default to `n`, please remove this line.

Reply:
Remove also the # Hidden option line